src/test/java/org/magicdgs/readtools/utils/fastq/FastqReadNameEncodingUnitTest.java

```diff
@@ -109,6 +109,31 @@
                         "@ST-E00169:175:HMTL3CCXX:7:1101:3457:1573 2:Y:0:NTGATTAC",
                         "@ST-E00169:175:HMTL3CCXX:7:1101:3457:1573", false, true, true,
                         new String[] {"NTGATTAC"}},
+                // casava with numbers instead of barcodes
+                {FastqReadNameEncoding.CASAVA,
+                        "@EAS139:136:FC706VJ:2:2104:15343:197393 1:N:18:1",
+                        "@EAS139:136:FC706VJ:2:2104:15343:197393", true, false, false,
+                        new String[] {"1"}},
+                // first of pair Casava format dual indexing with + separator (found in at least one provider)
+                {FastqReadNameEncoding.CASAVA,
+                        "@E00514:354:HLH2VCCXY:4:1101:21968:1784 1:N:0:CCCCCCCC+CCCCCCCC",
```

Comment:
Add another test case with a properly formatted barcode (with hyphen separator) to check if the split code also works with this kind of header (e.g., if someone changes the default separator).

Reply:
Done